root/edit/details/edit_label.tt

```diff
@@ -1,60 +0,0 @@
-[%- PROCESS 'edit/details/macros.tt' -%]
-
-<table class="details edit-label">
-  <tr>
-    <th>[% l('Label:') %]</th>
-    <td colspan="2">[% link_entity(edit.display_data.label) %]</td>
-  </tr>
-
-  [% display_diff(l('Name:'),
-                  html_escape(edit.display_data.name.old),
-                  html_escape(edit.display_data.name.new),
-                  '\s+') %]
-
-  [% display_diff(l('Sort name:'),
-                  html_escape(edit.display_data.sort_name.old),
-                  html_escape(edit.display_data.sort_name.new),
-                  '\s+') %]
```

Comment:
You completely dropped this. Labels no longer have sort names, but are you sure it's never needed for historical data?